Libraries/Network/NetInfo.js

```diff
@@ -79,56 +80,51 @@ const _isConnectedSubscriptions = new Map();
  * NetInfo exposes info about online/offline status
  *
  * ```
- * NetInfo.fetch().then((reach) => {
- *   console.log('Initial: ' + reach);
+ * NetInfo.getConnection().then((connection) => {
```

Comment:
@shergin I modeled it after the W3C Network Information API which offers this information off of a `connection` property (https://wicg.github.io/netinfo/#-dfn-connection-dfn-attribute). I can switch it to `getConnectionInfo` if you'd prefer.